source/content/site-dashboard.md

```diff
@@ -57,11 +58,21 @@ This environment is for your live site, where content creators actively post and
 
 ## Left Navigation
 
-The following links are located within your Dev, Test, and Live environment tabs:
+The following buttons are found above the lefthand navigation menu on every environment in the site dashboard:
 
-- **Code:** Switch between SFTP Connection Mode and Git Connection Mode. Obtain SFTP Connection Info, and commit changes to your Dev environment. (This link is only accessible within your Dev environment.)
+- **Visit `Environment` Site**: Where `Environment` is a placeholder for the active environment you're currently looking at in the dashboard, such as **Visit Test Site** or **Visit Development Site** etc. This will open the homepage of your site for the given environment in a new tab.
 
-- **Commit Log:** Review a list of all code changes by viewing the commits. Click an individual commit to see the log message and list of changes.
+- **Site Admin**: This will open your sites admin login page for the given environment in a new tab. If you're not already logged into your sites admin interface, this button will open `/user/login` for Drupal and `/wp-login.php` for WordPress. Once you've logged in and have an active admin session, this button will take you to `/user` for Drupal and `/wp-admin` for WordPress.
+
+<Alert title="Note" type="info">
+
+By default, the buttons described above will open links that are served via their respective [platform domains](/guides/domains/platform-domains) (e.g., `test-example.pantheonsite.io`). If you want these buttons to open your site using your own custom domain instead (e.g., `test.example.com`) you can connect a [custom domain](/guides/domains/custom-domains) then configure it as the [primary domain](/guides/domains/primary-domain) for the target environment. This is typically only done for the Live environment as part of launch however you can do this anytime for any environment based on your preferences and requirements.
```

Comment:
Probably not worth overloading this note but WordPress multisites break if you set a platform domain, so this can't actually be done (or shouldn't) for WordPress multisites that host multiple domains (even for live environments).

Reply:
We could warn about that in the primary domain doc that's crosslinked in this note? https://docs.pantheon.io/guides/domains/primary-domain

Reply:
@jazzsequence can you review the latest commit when convenient? https://github.com/pantheon-systems/documentation/pull/9342/commits/a2190f3a9f7ac684cd2a39a97eaff896e351eb73